util/syn_yosys.sh

```diff
@@ -10,59 +10,139 @@
 # The following tools are required:
 #  - sv2v: SystemVerilog-to-Verilog converter from github.com/zachjs/sv2v
 #  - yosys: synthesis tool from github.com/YosysHQ/yosys
+#  - Cadence Conformal
 #
 # Usage:
-#   syn_yosys.sh 2>&1 | tee syn.std
+#   ./syn_yosys.sh |& tee syn.std
 
 #-------------------------------------------------------------------------
 # use fusesoc to generate files and file list
 #-------------------------------------------------------------------------
-\rm -Rf build
-fusesoc --cores-root .. run --target=sim --setup --build formal > /dev/null 2>&1
+rm -Rf build syn_out
+
+fusesoc --cores-root .. run --tool=icarus --target=syn \
+  --setup "lowrisc:systems:top_earlgrey" > /dev/null 2>&1
```

Comment:
Why do you specify `--tool=icarus` here? I can believe there's a good reason but, if so, it probably needs an explanatory comment.

Reply:
Good point, I'll use what the latest version of syn_yosys.sh is using.